plugins/api-docs/README.md

```diff
@@ -15,6 +15,7 @@ Right now, the following API formats are supported:
 - [OpenAPI](https://swagger.io/specification/) 2 & 3
 - [AsyncAPI](https://www.asyncapi.com/docs/specifications/latest/)
 - [GraphQL](https://graphql.org/learn/schema/)
+- [gRPC](https://grpc.io/) - Using [GenDocu](https://github.com/pseudomuto/protoc-gen-doc) if configured, otherwise plain text.
```

Comment:
GenDocu and `protoc-gen-doc` are separate things, developed independently. Perhaps this should mention both, e.g.:

> - [gRPC](https://grpc.io/) - Using [GenDocu](https://github.com/gendocu-com/grpc-docs/) and [protoc-gen-doc](https://github.com/pseudomuto/protoc-gen-doc) plugins if configured, otherwise plain text.